common/test/test_snapshots.py

```diff
@@ -226,7 +226,7 @@ def test_rsyncSuffix(self):
                                                        '*blub',
                                                        '/bar/2'])
         self.assertRegex(suffix, r'^ --chmod=Du\+wx  ' +
-                                 r'--exclude="/tmp/.*?" ' +
+                                 r'--exclude=".*?\.local/share/backintime/mnt.*?" ' +
```

Comment:
The mountpoint is already excluded with next line. But the destination `/tmp/...` must stay excluded here. Otherwise we would backup snapshots again